tests/test_backends.py

```diff
@@ -176,6 +176,16 @@ def test_backend_methods(backend):
         np.array([2, 1, 0]),
         atol=1e-5,
     )
+    np.testing.assert_allclose(
+        tc.backend.copy(arr),
+        arr,
+        atol=1e-5,
+    )
+    np.testing.assert_allclose(
+        tc.backend.power(arr,6),
+        np.power(arr,6),
+        atol=1e-5
+    )
```

Comment:
could these two methods tests also move to ``test_backend_method_2``? namely, whether currently pytorch backend supports these two methods

Reply:
yes,change will be committed soon